checker/checker.go

```diff
@@ -429,7 +432,9 @@ func (v *visitor) MemberNode(node *ast.MemberNode) (reflect.Type, info) {
 		return t, info{}
 
 	case reflect.Array, reflect.Slice:
-		if !isInteger(prop) && !isAny(prop) {
+		if isInteger(prop) || isAny(prop) {
```

Comment:
Let's keep original if and do not flip condition.

Reply:
Ok.